include/SFML/Window/WindowBase.hpp

```diff
@@ -185,20 +185,18 @@ class SFML_WINDOW_API WindowBase
     /// thus you should always call this function in a loop
     /// to make sure that you process every pending event.
```

Comment:
This paragraphs needs to be updated, not to reference the former event argument.
Comment applies to waitEvent documentation too.